llvm/docs/Contributing.rst

```diff
@@ -201,5 +182,6 @@ of LLVM's high-level design, as well as its internals:
 .. _bug tracker: https://github.com/llvm/llvm-project/issues
 .. _clang-format-diff.py: https://reviews.llvm.org/source/llvm-github/browse/main/clang/tools/clang-format/clang-format-diff.py
 .. _git-clang-format: https://reviews.llvm.org/source/llvm-github/browse/main/clang/tools/clang-format/git-clang-format
-.. _LLVM's Phabricator: https://reviews.llvm.org/
+.. _LLVM's GitHub: https://github.com/llvm/llvm-project
+.. _LLVM's Phabricator (soon deprecated): https://reviews.llvm.org/
```

Comment:
Just call it "deprecated", as basically it's already is.

Reply:
Done.